rabbit-escape-render/src/rabbitescape/render/Polygon.java

```diff
@@ -0,0 +1,45 @@
+package rabbitescape.render;
+
+import java.util.Vector;
+
+import rabbitescape.engine.util.Util;
+
+public class Polygon
+{
+    public Vector<Integer> x, y;
```

Comment:
I want to change to float from int too. I had some half working code that smoothed the water surface, and that would look better with float.